orchestration/dags/data_gcp_dbt/models/intermediate/firebase/native/int_firebase__native_event.sql

```diff
@@ -90,8 +104,9 @@ SELECT
     CASE WHEN event_name = "screen_view" AND firebase_screen IN ("Bookings","BookingDetails") THEN 1 ELSE 0 END AS is_screen_view_bookings,
     CASE WHEN firebase_screen = "SignupConfirmationEmailSent" OR event_name = "ContinueCGU" THEN 1 ELSE 0 END AS is_signup_completed,
     CASE WHEN firebase_screen IN ("BeneficiaryRequestSent","UnderageAccountCreated","BeneficiaryAccountCreated") THEN 1 ELSE 0 END AS is_benef_request_sent,
-    CASE WHEN event_name = "login" THEN 1 ELSE 0 END AS is_login
-FROM {{ ref("int_firebase__native_event_flattened") }}
+    CASE WHEN event_name = "login" THEN 1 ELSE 0 END AS is_login,
+FROM {{ ref("int_firebase__native_event_flattened") }} AS e
+LEFT JOIN {{ ref("int_firebase__native_attribution") }} AS a ON a.user_pseudo_id = e.user_pseudo_id AND e.event_timestamp >= a.campaign_begins_at AND e.event_timestamp < a.campaign_ends_at
```

Comment:
jointure plus nécessaire après le 4/4/2024 ==> on pourrait l'ajouter dans un bloc jinja {% if ds() > date_debut and ds() < date_fin %}
OU
run le modèle en one shot qui écraserait les partitions concernées et donc pas besoin de jointure

Reply:
Oui je suis d'accord ! Juste à voir si jamais on décide d'appliquer le modèle d'attribution à toutes les données.